packages/liveblocks-react/src/umbrella-store.ts

```diff
@@ -717,6 +725,39 @@ export class UmbrellaStore<M extends BaseMetadata> {
     };
   }
 
+  /**
+   *
+   */
+  public getRoomInboxNotificationsLoadingState(
+    roomId: string
+  ): InboxNotificationsAsyncResult {
+    const queryKey = makeRoomInboxNotificationsQueryKey(roomId);
+
+    const resource = this._roomNotifications.get(queryKey);
+    if (resource === undefined) {
+      return ASYNC_LOADING;
+    }
+
+    const asyncResult = resource.get();
+    if (asyncResult.isLoading || asyncResult.error) {
+      return asyncResult;
+    }
+
+    const page = asyncResult.data;
+    // TODO Memoize this value to ensure stable result, so we won't have to use the selector and isEqual functions!
+    return {
+      isLoading: false,
+      // FIXME: The filtering part doesn't look right, should I created a notificationsDB like threadsDB?
+      inboxNotifications: this.getFullState().cleanedNotifications.filter(
```

Comment:
The filtering part doesn't look right, should I create a notificationsDB like threadsDB?